wee_slack.py

```diff
@@ -1021,14 +1037,34 @@ def should_try(self):
     def retry_ready(self):
         return (self.start_time + (self.tries**2)) < time.time()
 
+class SlackSubteam(object):
+   """
+   Represents a slack group or subteam 
+   """
+   def __init__(self, originating_team_id, **kwargs):
+       self.handle = kwargs.get('handle', None)
+       self.identifier = kwargs['id']
+       self.name = kwargs.get('name', None) 
+       self.team_id = originating_team_id 
+
+   def __repr__(self):
+       return "Name:{} Identifier:{}".format(self.name, self.identifier)
+
+   def __eq__(self, compare_str):
+       if compare_str == self.subteam_id :
+           return True
+       else:
+           return False
```

Comment:
Just do:
```
return compare_str == self.subteam_id
```

Reply:
Yeah not sure why I did that. I will fix that

Reply:
You mean remove if/else from that too? Sure, go ahead.

Though, when I think about it, these methods don't really make sense. `__eq__` should check if it equals another instance of the same class, not compare with a string. I wonder if they are used anywhere.